datafusion/core/tests/fifo.rs

```diff
@@ -85,12 +85,7 @@ mod unix_test {
         fifo_path: &Path,
         with_unbounded_execution: bool,
     ) -> Result<SessionContext> {
-        let config = SessionConfig::new()
-            .with_batch_size(TEST_BATCH_SIZE)
-            .set_u64(
-                "datafusion.execution.coalesce_target_batch_size",
```

Comment:
This was missed in #4757 the new config returns an error for invalid keys and so caught this